dimagi/utils/django/email.py

```diff
@@ -30,6 +32,12 @@ def send_HTML_email(subject, recipient, html_content, text_content=None,
                          "\n {}".format(html_content)
             )
 
+    if ga_track:
+        url = "https://www.google-analytics.com/collect?v=1&tid={ga_tid}&cid={ga_cid}&t=event&ec=email".format(
+            ga_tid=settings.ANALYTICS_IDS['GOOGLE_ANALYTICS_ID'],
+            ga_cid=uuid.uuid4().hex)
+        new_content = '<img src="{url}&ea=open"/>\n</body>'.format(url=url)
+        html_content = html_content.replace('</body>', new_content)
```

Comment:
do we care about error handling here? I think if the number of `</body>` tags is anything besides 1 the behavior of this would be silently wrong. might be worth adding an assertion to verify this is always working as expected.